plugins/lookup/passwordstore.py

```diff
@@ -34,8 +34,8 @@
         default: 'no'
       umask:
         description:
-          - Sets the umask for the created .gpg files. The first octed must be greater than 3 (user readable). Note
-            pass' default value is '077'.
+          - Sets the umask for the created .gpg files. The first octed must be greater than 3 (user readable).
+          - Note pass' default value is C(077).
```

Comment:
It seems that this is interpreted as integer. The documentation is unclear as to whether quoting this would display the description ``as expected``.

Reply:
Then write `C('077')`.